docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -492,6 +492,86 @@ Create a directory at a specific path:
 - **name:** name of the directory
 - **path:** specific path where user wants to create the directory
 
+### Create File
+
+Create a file at a specific path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/touch
+    
+- **name:** name of the file
+- **path:** specific path where user wants to create the file
+
+### Rename
+
+Rename a file or directory:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/rename
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** path after renaming the selected file or directory
+
+### Move
+
+Move a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/move
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** target path for moving the selected file or directory
+
+### Copy
+
+Copy a file or directory to a destination path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/copy
+    
+- **src_path:** path of the selected file or directory
+- **dest_path:** target path for copying the selected file or directory
+
+**Note:** For **Apache Ozone** filesystem, copy operation returns a string of skipped files if their size is greater than configured chunk size.
+
+### Get Content Summary
+
+Fetch content summary for a specific file in **HDFS** or **Apache Ozone**:
+
+    curl -X GET https://demo.gethue.com/api/v1/storage/content_summary=/user/hue/weblogs.csv
+
+    curl -X GET https://demo.gethue.com/api/v1/storage/content_summary=ofs://ozone1/testvolume/testbucket/testfile.csv
+
+### Delete
+
+Delete a file or directory:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/rmtree
+    
+- **path:** path of the selected file or directory
+- **skip_trash:** boolean value to determine moving deleted file or directory to trash or not
+
+**Note:** `skip_trash` field is currently supported only for HDFS.
+
+### Set Replication
+
+Set the replication factor for a file in **HDFS**:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/set_replication
+    
+- **src_path:** path of the selected file
+- **replication_factor:** numerical value for setting the replication factor for the file
+
+### Restore Trash
+
+Restore a specific file or directory from trash in **HDFS**:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/trash/restore
+    
+- **path:** path of the specific file or directory to restore from trash directory
```

Comment:
path of the specific file or directory that you want to restore from the trash directory